pkg/awsvpc/parser.go

```diff
@@ -290,8 +290,11 @@ func (rc *AWSresourcesContainer) getSGconfig(
 		if err != nil {
 			return err
 		}
-		sgName := getResourceName(sg.Tags, *sg.GroupId)
-		commonvpc.NewSGResource(sgName, *sg.GroupId, vpc, NewAWSSGAnalyzer(sg), sgMap, sgLists)
+		sgName := *sg.GroupId
```

Comment:
write a method that get *types.SecurityGroup
and return the sg name.
use it here, and in 
(sga *AWSSGAnalyzer) Name()